inner-loop-scripts/shell/1-cluster-stamp.sh

```diff
@@ -74,7 +74,7 @@ kubectl wait --namespace cluster-baseline-settings --for=condition=ready pod --s
 
 ACR_NAME=$(az deployment group show -g $RGNAMECLUSTER -n cluster-0001 --query properties.outputs.containerRegistryName.value -o tsv)
 # Import ingress controller image hosted in public container registries
-az acr import --source docker.io/library/traefik:2.2.1 -n $ACR_NAME
+az acr import --source docker.io/library/traefik:v2.4.8 -n $ACR_NAME
```

Comment:
YOu need to do this in `08-secret-managment-and-ingress-controller.md` as well, right?

Reply:
Yes. Good catch. Thanks. Fixed

Reply:
coo!